escape_roomba/context_test.py

```diff
@@ -0,0 +1,62 @@
+"""Unit tests for context.Context."""
+
+import logging
+
+import pytest
+
+from escape_roomba.context import Context
+
+
+@pytest.mark.asyncio
+async def test_add_listener(mocker, event_loop):
+    context = Context()
+    context.logger = logging.getLogger('test')
+    context.client = mocker.Mock()  # No hair needed for this test.
```

Comment:
No... _hair_? What does that mean?

Reply:
I meant no extra mock properties, but it doesn't matter now since I've changed it to use the standard fixture!

Reply:
Got it. Thanks.